### ISSUE TITLE
Add Services to Allow Direct REST Commands to be sent

### ISSUE THREAD

**Issue by shbatm** [shbatm/hacs-isy994#24]
Need to add services to allow:
[ ] Fast On / Fast Off Commands
[ ] Fade Up/Down/Stop Commands
[ ] Scene DIM/BRIGHTEN (3% bump) Commands
[ ] Generic ISY Device Commands
[ ] Live Reload of ISY Nodes (disconnect, update, reconnect, reprocess nodes) so that Home Assistant doesn't need to be restarted every time something is changed in the ISY.

**Comment by bkvargyas** [comment]
Hello!   I just found this repository.   I just got started with HASS.IO, I Have 3 ISY99i's spread with over 300 Insteon and Z-Wave devices.   I'm staring the process of moving my ISY automations to node-red through HA.  I make extensive use of the Fast ON/Off in my programs, so I see this is an upcoming enhacement.    Being new to HA, I would rather keep myself away from YAML as much as possible, so I'm using your Integration.   Going to keep an eye out on this service thread :-)
